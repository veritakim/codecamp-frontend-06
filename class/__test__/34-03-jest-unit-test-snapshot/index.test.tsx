import { render } from "@testing-library/react"
import JestUnitSnapShotTestPage from "../../pages/34-03-jest-unit-snapshot"

it("컴포넌트가 기존이랑 바뀐게 없는지 비교해보기 - 스냅샷 테스트", () => {
  const result = render(<JestUnitSnapShotTestPage />)
  expect(result.container).toMatchSnapshot()
})