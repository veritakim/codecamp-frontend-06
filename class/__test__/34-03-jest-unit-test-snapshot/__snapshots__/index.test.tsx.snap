// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`컴포넌트가 기존이랑 바뀐게 없는지 비교해보기 - 스냅샷 테스트 1`] = `
<div>
  <div>
    <div>
      유리는 5살 입니다.
    </div>
    짱구의 취미 입력하기: 
    <input
      type="text"
    />
    <button>
      짱구와 놀러가기
    </button>
  </div>
</div>
`;
